### PR TITLE
Weekly docs update

### PR DESCRIPTION
**Pull request by OzGav**
voice.md is still being drafted but I haven't yet added the links to it so should be ok to go through

### CODE REVIEW

File: docs/player-support/google-cast.md [hunk 1]
```diff
@@ -5,13 +5,14 @@ Music Assistant has full support for Google Cast based devices. This includes Go
...
-- After re-enabling a disabled speaker, it can take a while before the speaker is rediscovered, the process can be sped up by restarting Music Assistant.
-- It is not possible to group/sync cast players from within the MA interface (simply because an api does not exist for that).
+- If your Chromecast speakers are not auto detected or randomly unavailable then make sure that your Cast enabled speakers are on the same network/subnet as your Music Assistant server. Additionally, ensure that multicast traffic (more specifically mDNS) can travel freely as that is used for the discovery of players
+- After re-enabling a disabled speaker, it can take a while before the speaker is rediscovered, the process can be sped up by restarting Music Assistant
+- It is not possible to group/sync cast players from within the MA interface (simply because an api does not exist for that)
+- Google TV devices (not the AV dongles) are disabled by default
```
Comment (marcelveldt) [inline docs/player-support/google-cast.md:18]: as well as actual TV's - just anything that reports its model as a tv device

File: docs/player-support/index.md [hunk 2]
```diff
@@ -5,24 +5,19 @@ description: Information Relevant to all Player Providers
 
 # Player Providers
 
-For specific player provider information refer to the relevant section in this document. Most players are enabled by default and will be automatically discovered by Music Assistant.
+For specific player provider information refer to the relevant section in this document. Most players are enabled by default and will be automatically discovered by Music Assistant. The following table summarises player capabilities
```
[the remark this answers is not in the document]
Comment (OzGav) [reply docs/player-support/index.md:8]: Let me fix that. I thought I had found all of that change but clearly not!

File: docs/player-support/google-cast.md (hunk 1 above)
Comment (OzGav) [reply docs/player-support/google-cast.md:18]: I thought that and thus "Google TV devices" was my attempt to capture that idea. Thoughts on better words?

Comment (marcelveldt) [reply docs/player-support/google-cast.md:18]: Well the name of their latest stick is literally Google TV so maybe just refer to TV/Video devices here ?

Comment (OzGav) [reply docs/player-support/google-cast.md]: Ok done

**Review by jozefKruszynski — APPROVED** [review]

**Pull request merged**